### PR TITLE
183730516 - add pythnet in views

### PR DESCRIPTION
**Pull request by kamtarnowski**
#### What's this PR do?
PR applies the pythnet to views.

#### How should this be manually tested?
Things that having impact due to ticket (please test them):
- commision histories (index)
- stake account (index)
- ping things (index)
- check homepage if data is correct (switch between networks)

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183730516)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

**Review by maciejkocylapc — APPROVED** [review]

File: app/views/devise/shared/_side_banner.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <li>
     <i class="fas fa-long-arrow-alt-right text-purple me-2" aria-hidden="true"></i> Watchlist
     <div class="text-muted ms-4 ps-1">
-      Create your own list of favourite validators for mainnet, testnet, or both.
+      Create your own list of favourite validators for either mainnet, testnet, pythnet or for all of them.
```
Comment (kbiala) [inline app/views/devise/shared/_side_banner.html.erb:13]:
```suggestion
      Create your own list of favourite validators for mainnet, testnet or pythnet.
```

File: test/javascript/commission_histories/index.test.js [hunk 2]
```diff
@@ -21,19 +21,16 @@ var mock_commission_changes = function(){
...
     const wrapper = shallowMount(IndexTemplate, {
-      propsData: {
-        network: 'testnet'
-      },
+      store,
       stubs: {BPagination: BPagination}
     })
 
-    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/testnet?")
+    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/mainnet?")
```
Comment (kbiala) [inline test/javascript/commission_histories/index.test.js:33]: Why do we change to mainnet in tests?

File: test/javascript/commission_histories/index.test.js [hunk 3]
```diff
@@ -43,14 +40,14 @@ describe('index_template', ()=> {
...
       propsData: {
-        query: 'abc',
-        network: 'testnet'
+        query: 'abc'
       },
       stubs: {BPagination: BPagination}
     })
 
-    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/testnet?query=abc&")
+    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/mainnet?query=abc&")
```
Comment (kbiala) [inline test/javascript/commission_histories/index.test.js:50]: Same here

**Comment by kbiala** [comment]
See questions about tests.
Othewise, looks and works good.

File: test/javascript/commission_histories/index.test.js (hunk 2 above)
Comment (kamtarnowski) [reply test/javascript/commission_histories/index.test.js]: network comes from Vuex's getter. I managed to set this up in recent commit.

File: test/javascript/commission_histories/index.test.js (hunk 3 above)
Comment (kamtarnowski) [reply test/javascript/commission_histories/index.test.js]: Reverted to testnet.

File: app/javascript/packs/commission_histories/index_template.vue [hunk 4]
```diff
@@ -45,27 +45,26 @@
...
         page: 1,
         total_count: 0,
         sort_by: 'created_at_desc',
-        api_url: api_url,
+        api_url: null,
         account_name: this.query
       }
     },
     created () {
+      this.api_url = this.query && !this.query == ''
```
Comment (kbiala) [inline app/javascript/packs/commission_histories/index_template.vue:65]: This is waaay to long condition for one-liner syntax (? :). I know it's not exactly in the scope of this story, but you could also fix that.

**Review by kbiala — APPROVED** [review]

**Pull request merged**